.changeset/calm-scenes-render.md

```
@@ -0,0 +1,7 @@
+---
+'foldkit': minor
+---
+
+Make negated Scene property and state assertions require their target element to exist. Use `toBeAbsent()` or `not.toExist()` when absence is the intended assertion.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)